modules/notes/assets/js/editor/module.js

```diff
@@ -0,0 +1,47 @@
+class Module extends elementorModules.editor.utils.Module {
+	onInit() {
+		this.contextMenuNotesGroup();
+	}
+
+	contextMenuNotesGroup() {
+		const elTypes = [ 'widget', 'section', 'column', 'container' ];
+
+		elTypes.forEach( ( type ) => {
+			elementor.hooks.addFilter( `elements/${ type }/contextMenuGroups`, this.contextMenuAddGroup );
+		} );
+	}
+
+	/**
+	 * Append the 'Notes' context menu group
+	 *
+	 * @since 3.8.0
+	 *
+	 * @param {Array} groups
+	 * @return {Array} The updated groups.
+	 */
+	contextMenuAddGroup( groups ) {
+		const deleteGroup = _.findWhere( groups, { name: 'delete' } );
+		let afterGroupIndex = groups.indexOf( deleteGroup );
+
+		if ( -1 === afterGroupIndex ) {
+			afterGroupIndex = groups.length;
+		}
+
+		groups.splice( afterGroupIndex, 0, {
+			name: 'notes',
+			actions: [
+				{
+					name: 'open_notes',
+					title: __( 'Notes', 'elementor' ),
+					shortcut: '<i class="eicon-pro-icon" />',
```

Comment:
```suggestion
					shortcut: '<i class="eicon-pro-icon"></i>',
```

Closing tag is required:
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element

Reply:
Done